episodes/4-minimal-reproducible-code.Rmd

```diff
@@ -147,6 +196,12 @@ prop_spectab <- control_spectab %>%
   mutate(prop = total_count/sum(total_count)) %>%
   filter(species_id == "DS") # keep only spectabilis
 
+### **EDIT-start ###
+
+# these proportions don't look right to me... not in the way that I was thinking about them at least...
```

Comment:
Can you explain what you mean by not right? How were you thinking about them?

Reply:
HA! That's hilarious, a thoroughly unhelpful comment on my part, I think I was thinking "let's discuss at our next meeting." I don't really remember right now without looking at it... Let me see if it makes sense when I look at the rendered website and if not remind me to get back to it! I think it was a question of how the proportions were being calculated compared to what I had done... I need to look at the outputs again